src/lorarmesh.cpp

```diff
@@ -14,38 +29,71 @@ LoraMesh::LoraMesh(DataQueue<message_t>* txQueue,
   this->txQueue = txQueue;
   this->rxQueue = rxQueue;
   this->router = router;
+  
+  Layer1 = new Layer1Class();
+  Layer1->setPins(LORA_CS, LORA_RST, LORA_IRQ);
+  Layer1->setTxPower(TX_POWER);
+  Layer1->setLoRaFrequency(LORA_FREQ);
+  int newTry = 0;
+  while (!Layer1->init() or newTry < 30) 
+  {
+    newTry++;
+  }
+  if (newTry < 30){
+    LL2 = new LL2Class(Layer1);  
+    LL2->setInterval(10000); // set to zero to disable routing packets
+    LL2->init();
+  }else{
+    delay(500);
+    ESP.restart();
```

Comment:
Take a look on how this is usually done ([Click here](https://github.com/Instituto-Atlantico/mesh-connect/blob/master/src/gateway.cpp#L45))

Reply:
it's done, but i implemented this way

Reply:
while (!Layer1->init()) {
    delay(CONNECT_DELAY_MILLS);
    if (attempts-- == 0)
      ESP.restart();
  }

Reply:
Nice!